lms/djangoapps/grades/tests/test_scores.py

```diff
@@ -47,20 +51,20 @@ class TestGetScore(TestCase):
     display_name = 'test_name'
     location = 'test_location'
 
-    SubmissionValue = namedtuple('SubmissionValue', 'exists, weighted_earned, weighted_possible')
-    CSMValue = namedtuple('CSMValue', 'exists, raw_earned, raw_possible')
+    SubmissionValue = namedtuple('SubmissionValue', 'exists, points_earned, points_possible, created_at')
```

Comment:
Not a bad idea, but I'd like to get this PR merged separate of that, because it will either require some careful line drawing through the middle of the submissions app, or a migration of a probably large table, if we want to make the naming consistent in submissions.